benchmarks/perf-tool/add-filters-to-dataset.py

```diff
@@ -0,0 +1,172 @@
+import getopt
+import os
+import random
+import sys
+
+import h5py
+
+from osb.extensions.data_set import Context, HDF5DataSet
+
+"""
+Script builds complex dataset with additional attributes from exiting dataset that has only vectors. 
+Additional attributes are predefined in the script: color, taste, age. Only HDF5 format of vector dataset is supported.
+
+Script generates additional dataset of neighbours (ground truth) for each filter type. 
+
+Example of usage:
+    
+    create new hdf5 file with attribute dataset
+    add-filters-to-dataset.py ~/dev/opensearch/k-NN/benchmarks/perf-tool/dataset/data.hdf5 ~/dev/opensearch/datasets/data-with-attr True False
+    
+    create new hdf5 file with filter datasets
+    add-filters-to-dataset.py ~/dev/opensearch/k-NN/benchmarks/perf-tool/dataset/data-with-attr.hdf5 ~/dev/opensearch/datasets/data-with-filters False True
+"""
+
+class Dataset():
```

Comment:
Should this class be exported? If not, cant we prepend with "_" to ensure no conflicts with other data set classes?

Reply:
ack, will use "_"